### PR TITLE
BREAKING CHANGE: Added malls.json/json structure change

### PR DESCRIPTION
**Pull request by vicevirus**
#8 #9 
# Description
Added malls to malls.json. There is a little bit of change in structure, hence the 'Breaking Change' tag. This is the second pull  request due to a mistake I just realized. Kuala Lumpur malls are now populated properly.

### CODE REVIEW

**Comment by farhan-helmy** [comment]
salam @vicevirus boleh letak script scrape python tu sekali? buat satu folder script letak dalam tu

**Comment by vicevirus** [comment]
> salam @vicevirus boleh letak script scrape python tu sekali? buat satu folder script letak dalam tu

Done! Macam ni ya?

**Comment by farhan-helmy** [comment]
> > salam @vicevirus boleh letak script scrape python tu sekali? buat satu folder script letak dalam tu
> 
> Done! Macam ni ya?
yuuppp tq boss

**Comment by vicevirus** [comment]
> > > salam @vicevirus boleh letak script scrape python tu sekali? buat satu folder script letak dalam tu
> > 
> > 
> > Done! Macam ni ya?
> 
> yuuppp tq boss
Thanks juga bro. Video sangat membantu untuk saya belajar tentang collaboration on Git.

File: prisma/mall.json [hunk 1]
```diff
@@ -1,108 +1,123 @@
...
-                "ÆON AU2 (Setiawangsa) Shopping Centre",
-                "Datum Jelatek Mall",
-                "Endah Parade",
-                "KL East Mall",
-                "M3 Mall",
-                "Melawati Mall",
-                "ÆON Mall Wangsa Maju",
-                "Wangsa Walk Mall"
-            ]
+        "Federal Territories": {
```
Comment (farhan-helmy) [inline prisma/mall.json:3]: Please update seed.ts code if you want to change the structure of this

File: prisma/mall.json [hunk 2]
```diff
@@ -1,108 +1,123 @@
...
+            "district": {
+                "Kuala Lumpur": {
```
Comment (farhan-helmy) [inline prisma/mall.json:5]: The for loop code now is calling `kl` but here it is `Kuala Lumpur` so please update either one

File: prisma/mall.json
Comment (farhan-helmy) [inline prisma/mall.json:1]: Other part looks great, will add to the roadmap to add more mall seeder tq

**Review by farhan-helmy — CHANGES_REQUESTED** [review]

File: prisma/mall.json (hunk 1 above)
Comment (farhan-helmy) [reply prisma/mall.json:3]:
```suggestion
        "federal-territories": {
```

File: prisma/mall.json (hunk 2 above)
Comment (farhan-helmy) [reply prisma/mall.json:5]:
```suggestion
                "kuala-lumpur": {
```

File: prisma/mall.json [hunk 3]
```diff
@@ -1,108 +1,123 @@
...
-                "Endah Parade",
-                "KL East Mall",
-                "M3 Mall",
-                "Melawati Mall",
-                "ÆON Mall Wangsa Maju",
-                "Wangsa Walk Mall"
-            ]
+        "federal-territories": {
+            "district": {
+                "kuala-lumpur": {
```
Comment (farhan-helmy) [inline prisma/mall.json:5]:
```suggestion
                "kualaLumpur": {
```

File: prisma/mall.json [hunk 4]
```diff
@@ -1,108 +1,123 @@
...
-                "ÆON AU2 (Setiawangsa) Shopping Centre",
-                "Datum Jelatek Mall",
-                "Endah Parade",
-                "KL East Mall",
-                "M3 Mall",
-                "Melawati Mall",
-                "ÆON Mall Wangsa Maju",
-                "Wangsa Walk Mall"
-            ]
+        "federal-territories": {
```
[the remark this answers is not in the document]
Comment (vicevirus) [reply prisma/mall.json:3]: Okay no prob. Noted.

**Pull request merged**